docs/websites/apache/running-fastcgi-php-fpm-on-ubuntu-14.04-with-apache.md

```diff
@@ -0,0 +1,104 @@
+Intall PHP5, PHP-FPM and Configuring them for use with Apache
+-------------------------------
+PHP makes it possible to produce dynamic and interactive pages using your own scripts and popular web development frameworks. A request made with prefork-MPM PHP, consists of interpreting files or associated scripts. Each interpretation draws on elements and actions to be executed. With Prefork-MPM, a file is read in an identical and independent manner on each request, without taking any previous requests into account. As a result, when a visitor's actions on a website involve a request on the system, the PHP interpreter must be activated, be put in memory, list the required elements, find them and then check that the code is valid. Finally, when everything is ready, it executes the request.
+
+With PHP-FPM, the elements and instructions called on when a request is made get cached in memory, to be reused directly if the same request is made again. Less requests on the filer therefore means decreased average load on the Linode, and better availability of resources to carry out other tasks. Using PHP-FPM, all these operations will already be known and recorded by the server. Their interpretation is thus much faster, as is the page loading that follows. 
+
+
+1. Ubuntu includes packages for installing PHP5 with support for PHP5-FPM. From the terminal, issue the following command:
+
+        sudo apt install libapache2-mod-fastcgi php5-fpm php5
+
+2. We will now configure Apache to pass all requests for PHP files, with the _php_ file extension, to the PHP wrapper through FastCGI. Once PHP5 is installed we'll want to ensure that the PHP5-FPM module is enabled. Issue the following command to do this:
+
+        sudo a2enmod actions fastcgi alias
+        sudo a2enconf php5-fpm
+
+3. It is best to configure PHP-FPM to use UNIX sockets instead of TCP. To verify if this is already the case, issue the following command in your terminal:
+
+        sudo grep -E '^\s*listen\s*=\s*[a-zA-Z/]+' /etc/php5/fpm/pool.d/www.conf
```

Comment:
This RE is a little obtuse. 

You also have two step threes. Why not just remove the first step three, as the second step three and step four seem to sufficiently explain what needs doing.